src/js/libs/astro/coo.js

```diff
@@ -466,3 +466,37 @@ Numbers.toSexagesimal = function(num, prec, plus) {
 			return sign+Numbers.format(n, 1);
 	}
 }
+
```

Comment:
Is this method different from what is in aladinutils ?

Reply:
The sign was bothering me, and I was confused about the precision too. But maybe I should still use the `Numbers.toSexagesimal` ?

Reply:
Sorry my comment is for degreesToString that seems to be in aladinutils and in coo. Should we remove it from coo but keep it in aladinutils ?

Reply:
Ah I forgot to push coo.js when I moved the function into AladinUtils -_-